src/Bicep.Core.Samples/Files/LoopsIndexed_LF/main.symbolicnames.json

```diff
@@ -221,7 +221,7 @@
       "properties": {
         "resolutionVirtualNetworks": [
           {
-            "id": "[resourceId('Microsoft.Network/virtualNetworks', format('{0}-{1}', variables('vnetConfigurations')[copyIndex()].name, copyIndex()))]"
+            "id": "[resourceId('Microsoft.Network/virtualNetworks', format('{0}-{1}', variables('vnetConfigurations')[add(parameters('index'), 1)].name, add(parameters('index'), 1)))]"
```

Comment:
I've reviewed the changes to the symbolicnames output; this actually looks like the correct codegen - I suspect the incorrect codegen was introduced by the PR to remove `resourceInfo()`.